### PR TITLE
Fix: add support for CA certificate to ElasticSearch chart

### PR DESCRIPTION
**Pull request by Dlorite**

### CODE REVIEW

File: charts/elasticsearch-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -52,17 +52,10 @@ spec:
           {{ end }}
```
Comment (carlosadiegosysdig) [inline charts/elasticsearch-exporter/templates/deployment.yaml:52]: If they use 'http', they need the values "http, name and port" to create the ES_URI.
If they use the 'https', they need the value "secretUrl" to create the ES_URI.

Maybe we could just make them add the "secretUrl" in both cases to create the ES_URI (and they just create the url with or without authentication).

File: charts/elasticsearch-exporter/values.yaml [hunk 2]
```diff
@@ -42,9 +42,8 @@ url:
   http: http
   name: elasticsearch
   port: 9200
-
-secretName: elastic-config
-
+secretURL:
```
Comment (daviddetorres) [inline charts/elasticsearch-exporter/values.yaml:45]: I'd add comments to these new parameters, so users can use them directly from the helm chart.

File: charts/elasticsearch-exporter/README.md [hunk 3]
```diff
@@ -6,11 +6,50 @@ Visit [PromCat.io](https://promcat.io/apps/elasticsearch) for dashboards, alerts
...
+```
+```
+helm install -n sysdig-agent my-release ./charts/elasticsearch-exporter/ \
+  --set namespaceName="logging" \
+  --set workloadType="statefulset" \
+  --set workloadName="elasticsearch" \
+  --set secretURL="elastic-url-secret" \
+  --set secretTLS="elastic-tls-secret"
+```
+# Deprecated
```
Comment (daviddetorres) [inline charts/elasticsearch-exporter/README.md:48]: I'd remove this section and only leave updated info.

File: charts/elasticsearch-exporter/values.yaml (hunk 2 above)
Comment (daviddetorres) [reply charts/elasticsearch-exporter/values.yaml:45]: Ok, I see that this is covered in the Readme file.

**Pull request merged**